shared/tests/utility/math/filter/TestFilters.cpp

```diff
@@ -165,12 +165,12 @@ static const std::array<MeasurementType, 101> measurements = {
     MeasurementType(-1.08344978673423),   MeasurementType(-1.69966623692842),  MeasurementType(-0.777801698154316),
     MeasurementType(-1.57714784288996),   MeasurementType(-0.971672418368469)};
 
-static constexpr int number_of_particles = 1000;
-static constexpr double deltaT           = 0.05;
+static constexpr int num_particles = 1000;
```

Comment:
What's up with this change?

Reply:
shadowed global declaration from `ParticleFitler.hpp`

Reply:
why is there a global variable in `ParticleFilter.hpp`?

Reply:
There's not. There's a global variable here. The (non-global) variable in particlefilter.hpp shadows this global variable.

Reply:
If this is a constant, shouldn't it be `SCREAMING_SNAKE_CASE`? which should also solve the problem